cmd/can/main.go

```diff
@@ -21,15 +22,19 @@ func main() {
 	if config.Debug {
 		fmt.Printf("Reading API specification from \"%s\"\n", cfg.GetOpenAPIFilepath())
 	}
+	// TODO: What if JSON?
```

Comment:
Just some notes.
1. JSON is valid yaml
2. I don't think we should hide the schema in use from the logic to load it. Whether you have a separate function or use an enum like facility, it should be clear and deliberate.